calm/dsl/api/task.py

```diff
@@ -0,0 +1,6 @@
+from .resource import ResourceAPI
+
+
+class TaskAPI(ResourceAPI):
```

Comment:
Can we change the name to some `TaskLibraryApi` as they are used only in task library ?
(Maybe I am not correct enough.)

Reply:
done changed to TaskLibraryApi